src/realmd/AuthSocket.cpp

```diff
@@ -511,6 +513,13 @@ bool AuthSocket::_HandleLogonProof()
     if (!Read((char*)&lp, sizeof(sAuthLogonProof_C)))
         return false;
 
+    PINData pinData;
```

Comment:
That'd be my preference but I just followed the example set above by "sAuthLogonProof_C lp;".